docs/DeveloperGuide.md

```diff
@@ -339,6 +339,13 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * **Mainstream OS**: Windows, Linux, Unix, MacOS
 * **Private contact detail**: A contact detail that is not meant to be shared with others
+* **CCA Manager**: The name used to refer to this project
+* **CCA**: Refers to the student organisation which `CCA Manager` intends to simplify the administration of. 
+* **EXCO Member**: A target user group. Refers to a member that is in-charge of the organisation of a `CCA`.
+* **CCA Member**: A target user group. Refers to a member that partakes in the activities of a `CCA`.
+* **Contact**: An object that refers to a person. It is the core object of administration in **CCA Manager**. 
+* **Role**: Part of the metadata of a `contact`. It classifies a `contact` based on their responsibilities in the CCA.
+* **Group**: Part of the metadata of a `contact`. It allows `contact`s to be organised in groups the user can manage seperately.
```

Comment:
Might be better to use italics instead. Code format is for code, but italics is for special terms generall in Markdown
```suggestion
* **CCA**: Refers to the student organisation which *CCA Manager* intends to simplify the administration of. 
* **EXCO Member**: A target user group. Refers to a member that is in-charge of the organisation of a *CCA*.
* **CCA Member**: A target user group. Refers to a member that partakes in the activities of a *CCA*.
* **Contact**: An object that refers to a person. It is the core object of administration in **CCA Manager**. 
* **Role**: Part of the metadata of a *contact*. It classifies a *contact* based on their responsibilities in the CCA.
* **Group**: Part of the metadata of a *contact*. It allows *contact*s to be organised in groups the user can manage seperately.
```